.travis.yml

```diff
@@ -28,13 +28,9 @@ install:
   - conda info -a
 
   # create environment and install dependencies
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pip coverage
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION coveralls
```

Comment:
Is there a particular reason coveralls installed here and not in the yml file like all the other modules?

Reply:
My reasoning was that `coveralls` is not required on a developer's environment, *only* on travis (as it just sends the info from travis to coveralls.io)

Reply:
hm okay fair enough